lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -127,17 +128,21 @@ spec = do
             <> cmdKey
 
 
-    let shouldStdOut args expected = it (unwords args) $
+    let shouldStdOut args expected = it (unwords args) $ do
+            hSetEncoding IO.stdout utf8
+            hSetEncoding IO.stderr utf8
             case execParserPure defaultPrefs parser args of
                 Success x -> capture_ x >>= (`shouldBe` expected)
                 CompletionInvoked _ -> expectationFailure
                     "expected parser to show usage but it offered completion"
                 Failure failure ->
                     expectationFailure $ "parser failed with: " ++ show failure
-    let expectStdErr args expectation = it (unwords args) $
+    let expectStdErr args expectation = it (unwords args) $ do
+            hSetEncoding IO.stdout utf8
+            hSetEncoding IO.stderr utf8
```

Comment:
@jbgi you may want to use: https://input-output-hk.github.io/cardano-wallet/haddock/edge/cardano-wallet-launcher-2020.3.3/Cardano-Startup.html#v:withUtf8Encoding

Reply:
ah nice!

Reply:
`withUtf8Encoding` did not have any effect on the test failing (presumably due to `withCP65001` logic), had to use `setUtf8EncodingHandles` directly.